lib/_http_outgoing.js

```diff
@@ -558,7 +552,7 @@ OutgoingMessage.prototype._implicitHeader = function _implicitHeader() {
 Object.defineProperty(OutgoingMessage.prototype, 'headersSent', {
   configurable: true,
   enumerable: true,
-  get: function() { return !!this._header; }
+  get: () => !!this._header
```

Comment:
This modification will change the value of `this` and will cause `test-http-header-read` and `test-http-client-reconnect-bug` to fail. (Please run tests before submitting changes!)